modules/standard/ChapelIO.chpl

```diff
@@ -38,74 +38,41 @@ of a Hello World program:
  // outputs
  // Hello, World!
 
-.. _readThis-writeThis-readWriteThis:
+.. _readThis-writeThis:
 
-The readThis(), writeThis(), and readWriteThis() Methods
+The readThis() and writeThis() Methods
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. warning:: 'readWriteThis' methods are deprecated. Use 'readThis' and 'writeThis' methods instead.
-
-When programming the input and output method for a custom data type, it is
-often useful to define both the read and write routines at the same time. That
-is possible to do in a Chapel program by defining a ``readWriteThis`` method,
-which is a generic method expecting a single :record:`~IO.channel` argument.
-
-In cases when the reading routine and the writing routine are more naturally
-separate, or in which only one should be defined, a Chapel program can define
-``readThis`` (taking in a single argument - a readable channel) and/or
-``writeThis`` (taking in a single argument - a writeable channel).
-
-If none of these routines are provided, a default version of ``readThis`` and
-``writeThis`` will be generated by the compiler. If ``readWriteThis`` is
-defined, the compiler will generate ``readThis`` or ``writeThis`` methods - if
-they do not already exist - which call ``readWriteThis``.
+A Chapel program can implement ''readThis'' and ''writeThis'' methods on a
```

Comment:
These render a little oddly for me, are they two single quotes in a row?  Should they be double quotes?

Reply:
I pushed some changes that use double-backticks instead: ``readThis`` and ``writeThis``